src/middlewares/permissions.js

```diff
@@ -15,10 +17,10 @@ class PermissionMiddlewareCreator {
   }
 
   _getSmartActionInfoFromRequest(request) {
-    const smartActionEndpoint = request.originalUrl;
+    const smartActionEndpoint = `${request.baseUrl}${request.path}`;
     const smartActionHTTPMethod = request.method;
     const smartAction = Schemas.schemas[this.collectionName].actions.find((action) => {
-      const endpoint = action.endpoint || `/forest/actions/${parameterize(action.name)}`;
+      const endpoint = action.endpoint || `/actions/${parameterize(action.name)}`;
```

Comment:
Why remove the `/forest/` now that we're using `${request.baseUrl}${request.path}`?

Reply:
I don't recall why I got rid of it, I'll add it back